lib/pure/terminal.nim

```diff
@@ -908,6 +908,40 @@ proc newTerminal(): owned(PTerminal) =
   when defined(windows):
     initTerminal(result)
 
+
+runnableExamples:
```

Comment:
Same, this is not example code, that's a test. It also shows implementation details...